### PR TITLE
2022q4/core.adoc: Add report

### PR DESCRIPTION
**Pull request by lsalvadore**

### CODE REVIEW

**Comment by lwhsu** [comment]
core.adoc instead of code.adoc?

**Comment by lsalvadore** [comment]
Thanks @lwhsu. The typo was only in the commit message and in the title of the pull request.

Usually we do not force push commits, however since in this case review has just begun and only a small typo needed to be fixed in a commit message without touching any file, I made an exception.

**Comment by lsalvadore** [comment]
Please note that @grahamperrin suggested some changes in https://github.com/lsalvadore/freebsd-quarterly/pull/1 .

@sergio-carlavilla, @lwhsu: Could you please review those changes? Do you approve them? Shall we merge them? Thanks.

File: 2022q4/core.adoc
Comment (pauamma) [inline 2022q4/core.adoc:13]: s/present/present the draft/

**Review by pauamma — APPROVED** [review]
Single word missing, fixable while/after merging if you agree.

**Comment by sergio-carlavilla** [comment]
> Please note that @grahamperrin suggested some changes in [lsalvadore#1](https://github.com/lsalvadore/freebsd-quarterly/pull/1) .
> 
> @sergio-carlavilla, @lwhsu: Could you please review those changes? Do you approve them? Shall we merge them? Thanks.

Unless a core team member approve it, keep the document as I sent it please.

**Comment by pauamma** [comment]
> > Please note that @grahamperrin suggested some changes in [lsalvadore#1](https://github.com/lsalvadore/freebsd-quarterly/pull/1) .
> > @sergio-carlavilla, @lwhsu: Could you please review those changes? Do you approve them? Shall we merge them? Thanks.
> 
> Unless a core team member approve it, keep the document as I sent it please.

Deferring merge until either a core member opines on the change requests or all reports are in, whichever comes first.

**Comment by sergio-carlavilla** [comment]
No, make the commit as it. And if before you send the announcement some from core approves the changes, go ahead. If not, commit as it. This document was approved by core@.

@lsalvadore proceed with the commit please

**Comment by lsalvadore** [comment]
Merged with commit 55f79a31babe3aad87e3be4eb3001fecb188cc02 without any change.

I close the pull request manually since GitHub does not notice that I have merged it.

**Comment by lwhsu** [comment]
> Thanks @lwhsu. The typo was only in the commit message and in the title of the pull request.
> 
> Usually we do not force push commits, however since in this case review has just begun and only a small typo needed to be fixed in a commit message without touching any file, I made an exception.

I think that usually in a pull-reuqest workflow, force push to a feature branch is not a rare thing. It makes more sense to make the history (how a modification is separated into commits, the commit message) perfect in the public branches.

**Pull request closed without merging**
